src/views/SystemList/Table.tsx

```diff
@@ -31,10 +36,12 @@ import TableResults from 'components/TableResults';
 import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices'; // May be temporary if we want to hard code all the CMS acronyms.  For now it creates an acronym for all capitalized words
 import CreateCedarSystemBookmarkQuery from 'queries/CreateCedarSystemBookmarkQuery';
 import DeleteCedarSystemBookmarkQuery from 'queries/DeleteCedarSystemBookmarkQuery';
+import GetMyCedarSystemsQuery from 'queries/GetMyCedarSystemsQuery';
+// import { mapCedarStatusToIcon } from 'types/iconStatus';
```

Comment:
✂️ ?

Reply:
Yeah that was legacy commented out code, and wasnt unsure if I wanted to remove it.  Probably couldn't hurt, I think it's really old